<launch>
 <node pkg="motor_communication" type="motor_control_node" name="motor_control_node" output="screen">
    <param name="motor_serial_port" value="/dev/ttyUSB2" />
    <param name="motor_baud_rate" value="9600" />
    <remap from="motor_control/speed" to="manual_control/speed"/>
    <remap from="motor_control/stop_start" to="manual_control/stop_start"/>   
  </node>
</launch>
